contracts/Removal.sol

```diff
@@ -361,7 +340,7 @@ contract Removal is
     uint256[] memory ids,
     uint256[] memory amounts,
     bytes memory data
-  ) public override {
+  ) public override(IERC1155Upgradeable, ERC1155Upgradeable) {
```

Comment:
why are both the interface and the implementation contract in here?

Reply:
Was required